docs/AboutUs.md

```diff
@@ -35,18 +35,18 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 [[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
 
-* Role: Developer
-* Responsibilities: Data
+-   Role: Developer
+-   Responsibilities: Data
 
-### Jean Doe
+### Kwan Hao Wei
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/kwanhw.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/kwanhw)]
+[[portfolio](team/kwanhw.md)]
 
-* Role: Developer
-* Responsibilities: Dev Ops + Threading
+-   Role: Developer
```

Comment:
Please change to asterisk (for standardization across the file).

Reply:
Fixed in cb3b9c5